package.json

```diff
@@ -173,12 +173,15 @@
     "node": "^14.15.0 || ^16.10.0 || >=18.0.0"
   },
   "resolutions": {
+    "@jest/core": "file:./packages/jest-core",
```

Comment:
Turns out these are needed. The best way I've found to keep from `yarn.lock` churn is to:

```
make file changes, bump versions, etc
yarn # updates yarn.lock
git commit
yarn # updates yarn.lock hashes
```